src/pages/GamePage/GameWaitingRoom/GameReadyAndStart.tsx

```diff
@@ -1,6 +1,18 @@
-const GameReadyAndStart = () => {
+import { CompatClient } from '@stomp/stompjs';
+import storageFactory from '@/utils/storageFactory';
+
+const GameReadyAndStart = ({ ws }: { ws: CompatClient | null }) => {
+  const { getItem } = storageFactory(localStorage);
+
+  const token = getItem('token', '');
+  const connectHeaders = {
+    Authorization: 'Bearer ' + token,
```

Comment:
요 부분 백틱으로 바꾸면 일관성 있을 것 같아용! 👍
```suggestion
    Authorization: `Bearer ${token}`,
```